test/golden/Test.hs

```diff
@@ -54,7 +54,7 @@ map f [] = []
 map f (x : xs) = f x : map f xs
 
 mapTest :: [Integer] -> [Integer]
-mapTest = map (id . (+) 5)
+mapTest = map (id . (5 +))
```

Comment:
Given that you just added support for sections, the user could have written `map (id ∘ (5 +_))` in the Agda to get this compilation result. Why impose this behaviour, making it impossible to actually get `(+) 5`?

Reply:
The problem is that we cannot distinguish `_+_ 5` and `5 +_`, so I opted for always using the section.

Reply:
OK, this seems reasonable.